.github/workflows/compatibility-test.yml

```
@@ -0,0 +1,55 @@
+          composer require simplesoftwareio/simple-qrcode --dev
+          composer update --prefer-dist --no-interaction
+
+      - name: Execute tests
+        run: |
+          mkdir -p tests/Compatibility
+          cp .github/workflows/compatibility-test.php tests/Compatibility/SimpleQrCodeTest.php
+          ./vendor/bin/pest tests/Compatibility/SimpleQrCodeTest.php --colors=always
+
+      - name: Upload test results
+        if: ${{ failure() }}
+        uses: actions/upload-artifact@v3
```

actionlint (1.7.7)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)